PAWS/Home.swift

```diff
@@ -9,27 +9,34 @@
 import PAWSContacts
 import PAWSMockDataStorageProvider
 import PAWSSchedule
+import PAWSNotificationScreen
 import PAWSSharedContext
 import SwiftUI
 
 
 struct HomeView: View {
     enum Tabs: String {
-        case schedule
+        //case schedule
+        case notifications
         case contact
         case mockUpload
     }
     
     
-    @AppStorage(StorageKeys.homeTabSelection) var selectedTab = Tabs.schedule
+    @AppStorage(StorageKeys.homeTabSelection) var selectedTab = Tabs.notifications
     
     
     var body: some View {
         TabView(selection: $selectedTab) {
-            ScheduleView()
-                .tag(Tabs.schedule)
+            //            ScheduleView()
+            //                .tag(Tabs.schedule)
+            //                .tabItem {
+            //                    Label("SCHEDULE_TAB_TITLE", systemImage: "square.and.arrow.up")
+            //                }
```

Comment:
Please remove any code that is commented out to have a clean codebase:
```suggestion
            //            ScheduleView()
            //                .tag(Tabs.schedule)
            //                .tabItem {
            //                    Label("SCHEDULE_TAB_TITLE", systemImage: "square.and.arrow.up")
            //                }
```